checks.d/elastic.py

```diff
@@ -476,14 +497,13 @@ def _create_event(self, status):
 
         msg = "ElasticSearch: %s just reported as %s" % (hostname, status)
 
-        return { 'timestamp': int(time.time()),
-                 'event_type': 'elasticsearch',
-                 'host': hostname,
-                 'msg_text':msg,
-                 'msg_title': msg_title,
-                 'alert_type': alert_type,
-                 'source_type_name': "elasticsearch",
-                 'event_object': hostname
-            }
-
-
+        return {
+            'timestamp': int(time.time()),
+            'event_type': 'elasticsearch',
+            'host': hostname,
+            'msg_text': msg,
+            'msg_title': msg_title,
+            'alert_type': alert_type,
+            'source_type_name': "elasticsearch",
```

Comment:
Yeah i didn't change that parf of the code, just fixed the PEP8 violations. Fixing it